.ci/run-benchmark.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+
+# check if jar path is provided
+JAR_PATH=$1
+if [ -z "$JAR_PATH" ]; then
+    echo "Usage: $0 <target jar path>"
+    exit 1
+fi
+
+# Sample project path
+SAMPLE_PROJECT="./.ci/jdk22"
+
+# generate benchmark config file
+generate_config() {
+  local CONFIG="<?xml version=\"1.0\"?>
+<!DOCTYPE module PUBLIC \"-//Puppy Crawl//DTD Check
+Configuration 1.3//EN\"
+\"http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd\">
+<module name=\"Checker\">
+  <module name=\"TreeWalker\"/>
+</module>"
+  echo "$CONFIG" > ./.ci/benchmark-config.xml
+}
+
+# run a command and time it
+time_command() {
+  # Run the command with time
+  TEST_COMMAND=$1
+  # shellcheck disable=SC2086
+  TIME_OUTPUT=$(command time -p $TEST_COMMAND 2>&1)
+  # Extract execution time
+  EXECUTION_TIME=$(echo "$TIME_OUTPUT" | awk '/real/ {print $2}')
+
+  local RESULT_ARRAY=($EXECUTION_TIME)
+  echo "${RESULT_ARRAY[@]}"
+}
+
+# run the benchmark a few times to calculate the average metrics
+run_benchmark() {
+  local TOTAL_TIME=0
+  local NUM_RUNS=3
+
+  [ ! -d "$SAMPLE_PROJECT" ] &&
+    echo "Directory $SAMPLE_PROJECT DOES NOT exists." | exit 1
+
+  generate_config
+  for ((i = 1; i <= NUM_RUNS; i++)); do
+    echo "Running benchmark ${i}/${NUM_RUNS}..."
+    local CMD="java -jar $JAR_PATH -c ./.ci/benchmark-config.xml -x .git $SAMPLE_PROJECT"
+    local BENCHMARK=($(time_command "$CMD"))
+    TOTAL_TIME=$(echo "$TOTAL_TIME + ${BENCHMARK[0]}" | bc)
+    echo "================== BENCHMARK RESULT #${i} =================="
+    echo "Execution Time: ${BENCHMARK[0]} s"
```

Comment:
Remove all this output, just focus on accumulation time values to summ them.

Reply:
done